README.md

```diff
@@ -54,4 +54,47 @@ pelfs = PelicanFileSystem(“some-director-url”)
 file1 = PelicanMap(“/namespace/file/1”, pelfs=pelfs)
 file2 = PelicanMap(“/namespace/file/2”, pelfs=pelfs)
 ds = xarray.open_mfdataset([file1,file2], engine='zarr')
-```
\ No newline at end of file
+```
+
+### Specifying Endpoints
+
+The following describes how to specify endpoints to get data from, rather than letting PelicanFS and the director determine the best cache. PelicanFS allows you to specify whether to read directly from the origin (bypassing data staging altogether) or to name a specific cache to stage data into. 
+
+**Note**
+> If both direct reads and a specific cache are set, PelicanFS will use the specified cache and ignore the direct reads setting.
+
+
+#### Enabling Direct Reads
+
+Sometimes you might wish to read data directly from an origin rather than via a cache. To enable this at PelicanFileSystem creation, just pass in `dirReads=True` to the constructor.
+
+```python
+pelfs = PelicanFileSystem("some-federation-url", dirReads=True)
+```
+
+Direct reads can also be turned on an off using the class function `set_direct_reads` like so:
+
+```python
+pelfs.set_direct_reads(True)
```

Comment:
Let's drop the separate setter's; it doesn't seem commonly done in other implementations.